squad/api/rest.py

```diff
@@ -22,6 +43,27 @@ class ProjectViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return self.queryset.accessible_to(self.request.user)
 
+    @detail_route(methods=['get'])
+    def builds(self, request, pk=None):
+        builds = self.get_object().builds.order_by('-datetime')
+        page = self.paginate_queryset(builds)
+        serializer = BuildSerializer(page, many=True, context={'request': request})
+        return self.get_paginated_response(serializer.data)
+
+
+class BuildSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
I'm getting the following message when trying locally:
```
"Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the BuildSerializer serializer.",
```
It applies to builds, testjobs, testruns and environments.